src/main/constraints/87_vice_sessions.sql

```diff
@@ -0,0 +1,5 @@
+SET search_path = public, pg_catalog;
+
+ALTER TABLE ONLY "session"
+    ADD CONSTRAINT "session_pkey"
+    PRIMARY KEY ("sid") NOT DEFERRABLE INITIALLY IMMEDIATE;
```

Comment:
Our current naming standard for primary key constraint files would be `00_87_vice_sessions_pkey.sql` for this file.
Please see https://github.com/cyverse-de/de-db#constraints

Reply:
Sometimes I wonder if it would be clearer if we had a separate directory or a separate file dedicated to primary keys. 🤔

Reply:
A separate directory makes sense to me.

Reply:
I think it would require changes to https://github.com/cyverse-de/facepalm, though.

Reply:
Yeah, it would.